airbyte-integrations/connectors/source-launchdarkly/source_launchdarkly/launchdarkly.yaml

```diff
@@ -0,0 +1,114 @@
+      api_token: "{{ config['access_token'] }}"
+  offset_paginator:
+    type: DefaultPaginator
+    $options:
+      url_base: "*ref(definitions.requester.url_base)"
+    pagination_strategy:
+      type: "OffsetIncrement"
+      page_size: 20
+    page_token_option:
+      field_name: "offset"
+      inject_into: "request_parameter"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "limit"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      $ref: "*ref(definitions.offset_paginator)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  projects_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "projects"
+      primary_key: "_id"
+      path: "/projects"
+  project_stream_slicer:
+    type: SubstreamSlicer
+    parent_stream_configs:
+      - stream: "*ref(definitions.projects_stream)"
+        parent_key: key
+        stream_slice_field: project_key
+  environments_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "environments"
+      primary_key: "_id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        path: "/projects/{{ stream_slice.project_key }}/environments"
+      stream_slicer:
+        $ref: "*ref(definitions.project_stream_slicer)"
+  metrics_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "metrics"
+      primary_key: "_id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        path: "/metrics/{{ stream_slice.project_key }}"
+      stream_slicer:
+        $ref: "*ref(definitions.project_stream_slicer)"
+  members_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "members"
+      primary_key: "_id"
+      path: "/members"
+  audit_log_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "auditlog"
+      primary_key: "_id"
+      path: "/auditlog"
+  flags_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "flags"
+      primary_key: "key"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        path: "/flags/{{ stream_slice.project_key }}"
+      stream_slicer:
+        $ref: "*ref(definitions.project_stream_slicer)"
+
+streams:
+  - "*ref(definitions.projects_stream)"
+  - "*ref(definitions.environments_stream)"
+  - "*ref(definitions.metrics_stream)"
+  - "*ref(definitions.members_stream)"
+  - "*ref(definitions.audit_log_stream)"
+  - "*ref(definitions.flags_stream)"
```

Comment:
Are you planning to add the missing streams like Segments, Tags, Users? 
I saw the endpoint `audit_log` allow to sync incrementally, are you interested in implementing this sync mode?

Reply:
Yes, but not in this version, i prefer to stay with these streams as alpha version, and in couple month, adding other sync. modes and other streams.